### PR TITLE
Remove deep-extend dependency as it is not used in this package and…

### PR DESCRIPTION
**Pull request by mathew-fleisch**
…currently has a security vulnerability https://nvd.nist.gov/vuln/detail/CVE-2018-3750

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "swagger-parameters",
   "main": "index.js",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (bendrucker) [inline package.json]: Please do not touch the version here, that's my change to make via `npm version`

**Comment by bendrucker** [comment]
Also you shouldn't add a package lock to libraries. Took care of this directly.

File: package.json (hunk 1 above)
Comment (mathew-fleisch) [reply package.json]: thanks

Comment (mathew-fleisch) [reply package.json]: can you also please bump the version on swole? https://github.com/bendrucker/swole/blob/master/package.json#L48

Comment (bendrucker) [reply package.json]: By setting `~1.3.0`, swole allows patch versions. So I won't bump swole, it's not necessary or the correct way to handle semver in node. It's up to you as an application to opt into updates, including transitive ones (via your package-lock or shrinkwrap):

https://docs.npmjs.com/cli/update

**Pull request closed without merging**